src/components/Bar.jsx

```diff
@@ -218,9 +220,13 @@ export class Bar extends Component {
       ? { '--cozBarThemePrimaryContrastTextColor': pctColor }
       : {}
     const themeStyle = { ...pStyle, ...pctStyle }
+    const flagshipClass = isFlagshipApp ? ' coz-is-flagship-app' : ''
 
     return (
-      <div className={`coz-bar-wrapper coz-theme-${theme}`} style={themeStyle}>
+      <div
+        className={`coz-bar-wrapper coz-theme-${theme}${flagshipClass}`}
+        style={themeStyle}
+      >
```

Comment:
I would add one test, when isFlagship true, then we add correct class

Reply:
Alright